hack/verify-yamllint.sh

```diff
@@ -24,6 +24,8 @@ helmPath=charts/latest/csi-driver-nfs/templates
 echo "checking yaml files num ..."
 deployDirNum=`ls deploy/*.yaml | wc -l`
 helmDirNum=`ls $helmPath/*.yaml | grep -v serviceaccount | wc -l`
+# exclude 5 external-snapshotter resources that need to be part of the helm chart
+helmDirNum=$((helmDirNum-5))
```

Comment:
because https://github.com/kubernetes-csi/external-snapshotter/issues/812 is still open, I had a dilemma here. The `install-driver.sh` in https://github.com/kubernetes-csi/csi-driver-nfs/pull/430/commits/d369f94164b8587c0d53d9bce9e2569842924d1c handles external-snapshotter without the requirement to embed and copy/paste third party manifests.

I would like to try to tackle https://github.com/kubernetes-csi/external-snapshotter/issues/812, which would mean csi-driver helm charts will no longer need to embed the manifests and instead could just depend on the official external-snapshotter chart which would remove the need for this arithmetic offset.